### PR TITLE
Adding node support to readme

### LINKED ISSUE #1284: How to use highlight.js on nodejs
Hello,
I see on the homepage, hightlight.js can be used with nodejs as well. However I am not able to find any documentation or short sample which I can follow to use it on nodejs.
Would be really great if someone can point to right direction for this.

Thank you,

### PR DESCRIPTION
**Pull request by lukasoppermann**
This fixed #1284

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -287,6 +287,46 @@ onmessage = (event) => {
...
 ```
 
+## Node.js
+
+You can run highlight.js on node to prepare content before sending it to the browser.
+
+
+```js
+// require the highlight.js library including all languages
+const hljs = require('./highlight');
```
Comment (yyyc514) [inline README.md]: I know it's an example but should this be `highlight.js` just for consistency with the below example?

File: README.md [hunk 2]
```diff
@@ -287,6 +287,46 @@ onmessage = (event) => {
 };
 ```
 
+## Node.js
+
+You can run highlight.js on node to prepare content before sending it to the browser.
```
Comment (yyyc514) [inline README.md]: - run on or "use with"?  
- prepare vs "highlight"?

File: README.md [hunk 3]
```diff
@@ -287,6 +287,46 @@ onmessage = (event) => {
...
+```js
+// require the highlight.js library including all languages
+const hljs = require('./highlight');
+const highlightedCode = hljs.highlightAuto('<span>Hello World!</span>').value
+```
+
+```js
+// require the highlight.js library without languages
+const hljs = require("highlight.js/lib/highlight.js");
+// seperatly require languages
```
Comment (yyyc514) [inline README.md]: spelling: separately

File: README.md [hunk 4]
```diff
@@ -287,6 +287,46 @@ onmessage = (event) => {
...
+hljs.registerLanguage('html', require('highlight.js/lib/languages/html'));
+hljs.registerLanguage('sql', require('highlight.js/lib/languages/sql'));
+// highlight with providing the language
+const highlightedCode = hljs.highlight('html', '<span>Hello World!</span>').value
+```
+
+### highlight() return value
+```js
```
Comment (yyyc514) [inline README.md]: Should there be a line break here or does it really not matteR?

Comment (lukasoppermann) [reply README.md]: You mean above the code block, correct? I added one.

**Comment by egor-rogov** [comment]
I'm a bit concerned with "highlight() return value" and "highlightAuto() return value" subsections under "Node.js" section:

1. This information is not specific to Node.js
2. Such details are probably not intended for README, they are described in devveloper's docs : https://highlightjs.readthedocs.io/en/latest/api.html

What do you think?

**Comment by lukasoppermann** [comment]
Hey @egor-rogov, I understand the concern, however I feel that somehow we need to describe, why you have to use `.value` to get the result.

**Comment by yyyc514** [comment]
@egor-rogov Would this work better for you if we just dropped everything below:

"### highlight() return value"

I agree that's really for the API docs.  If a single example shows the use of `.value` I think that's clear enough at a glance, then if you want to know more, look at the docs.

File: README.md [hunk 5]
```diff
@@ -287,6 +287,48 @@ onmessage = (event) => {
 };
 ```
 
+## Node.js
+
+You can use highlight.js with node to highlight content before sending it to the browser.
```
Comment (lukasoppermann) [inline README.md]: How about adding the info here?

> You can use highlight.js with node to highlight content before sending it to the browser. Make sure to use the `.value` property to get the formatted html. For more info about the returned object refer to the api docs https://highlightjs.readthedocs.io/en/latest/api.html

Comment (egor-rogov) [reply README.md]: That's fine with me.

Comment (lukasoppermann) [reply README.md]: I updated it now.

**Comment by egor-rogov** [comment]
Great! Merged, thanks.

**Pull request merged**